tests/framework/Optimizers/Stochastic/stoch_parabola.py

```diff
@@ -17,7 +17,8 @@ def random(scale=0.5,loc=-1.0):
   return scale*(2.*np.random.rand()+loc)
 
 def evaluate(values):
-  return base(values) + random()
+  ran = random()/10.0
```

Comment:
Previously the test was converging; that was the point of the multiple gradient evaluations.  It was designed to closely mimic the 1:1 noise ratio.